lib/src/codegen.dart

```diff
@@ -51,6 +65,57 @@ String _baseExtendName(String extendName, Map<String, Element> allElements) {
   return baseExtendName;
 }
 
+_PropertyGenerator _propertyGenerator(Map<String,dynamic> classOverrides) {
+  if (classOverrides==null) {
```

Comment:
I would just do this on one line, and omit the curlies (same for _methodGenerator below)